### PR TITLE
Open issue with gh cli

### LINKED ISSUE #12: how to monitor health of the dashboard?
do we want to get notified if the table has 0 row?

### PR DESCRIPTION
**Pull request by Bisaloo**
Fix #12 

Tested in https://github.com/Bisaloo/testrepo

### CODE REVIEW

File: .github/workflows/render-dashboard.yml [hunk 1]
```diff
@@ -73,21 +73,8 @@ jobs:
...
 
-      - run: |
-          gh::gh(
-            "PATCH /repos/{repo}/issues/{issue_number}",
-            repo = "${{ github.repository }}",
-            issue_number = 53,
-            state = "open"
-          )
-        shell: Rscript {0}
+      - run: gh issue reopen 53
```
Comment (maelle) [inline .github/workflows/render-dashboard.yml:80]: this looks so simple!! :sparkles:

**Comment by maelle** [comment]
And really cool to see the example. Thank you! :rocket:

**Comment by llrs** [comment]
This is great! Is it documented on the actions/checkout ? (How did you find it?)

**Comment by Bisaloo** [comment]
I already suspected that `gh` was pre-installed in the containers used by GitHub Actions (it makes sense since both products are produced by the same company).

This idea was kind of reinforced by [this page in the official docs](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow) that was using `gh` without installing it explicitly.

So I ran some tests in my testrepo and confirmed it is indeed preinstalled. Then, it was just a matter of finding the right command in the [`gh` cli documentation](https://cli.github.com/manual/).

**Comment by llrs** [comment]
I somehow expected actions to be very lean. Thanks for the links!

**Pull request merged**